### DISCUSSION TITLE
Not able to save web deskstop

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#5693] [Q&A]
Dear all,

I'm trying to save a deskstop from the Web Portal, but I have this kind of error:

![Capture d’écran 2021-12-02 à 15 27 54](https://user-images.githubusercontent.com/6996265/144441413-2f2a7489-3133-4508-ab1a-cbcc48eb922e.png)

Maybe I missed some MySQL update?

Thank you

**Comment by fstagni** [comment]
Hi Luisa, which version of DIRAC server?

**Reply by arrabito** [reply]
7.3.7

**Reply by andresailer** [reply]
Seems like an old update was missed, see discussion in https://github.com/DIRACGrid/DIRAC/issues/2192
